src/routes/(console)/organization-[organization]/change-plan/+page.svelte

```diff
@@ -217,7 +217,7 @@
             await invalidate(Dependencies.ACCOUNT);
             await invalidate(Dependencies.ORGANIZATION);
 
-            await goto(`${base}/organization-${org.$id}`);
+            await goto(`${base}/organization-${org.$id}/billing`);
```

Comment:
I wouldn't redirect always to the billing tab... We could redirect to `previousPage` so they end up where they started :+1: 

(same comment applies to the other instance we do this)

Reply:
Fixed!